content/doc/book/blueocean/dashboard.adoc

```diff
@@ -19,105 +19,78 @@ endif::[]
 
 = Dashboard
 
-Blue Ocean's "Dashboard" is the default view shown when you open Blue Ocean and
-shows an overview of all Pipeline projects configured on a Jenkins instance.
+Blue Ocean's *Dashboard* is the default view shown when opening Blue Ocean.
+It disaplays an overview of all Pipeline projects configured on a Jenkins instance.
 
 include::doc/book/blueocean/_blue-ocean-status.adoc[]
 
-The Dashboard consists of a blue link:#navigation-bar[navigation bar] at the
-top, the link:#pipelines-list[Pipelines list], as well as the
-link:#favorites-list[Favorites list].
+The Dashboard consists of a blue link:#navigation-bar[navigation bar] at the top, the link:#pipelines-list[Pipelines list], and the link:#favorites-list[Favorites list].
```

Comment:
Or "the user interface"
```suggestion
Blue Ocean consists of a blue link:#navigation-bar[navigation bar] at the top, the link:#pipelines-list[Pipelines list], and the link:#favorites-list[Favorites list].
```

Reply:
same comment as above, leaving Dashboard terminology for consistency